app/pages/onboarding/03-restore-wallet/restore-wallet.tsx

```diff
@@ -40,11 +40,13 @@ export const RestoreWallet: React.FC = () => {
       return;
     }
 
-    if (!validateMnemonic(mnemonic)) {
+    const parsedMnemonic = mnemonic.toLowerCase().trim();
```

Comment:
wonder if it makes sense to add some additional fixes here since we're playing around with it? sometimes people have it with line breaks. It'd be nice to split the words and extract them into a new string that will always be valid. just a thought!